diff --git a/dist/types-BWScgZQh.mjs b/dist/types-BWScgZQh.mjs
index 87f4e9c1ebf45ae4f4dd6492f95a039da3de27b4..79bfa7aa7d3fe264c2107fe554d7cea13d667430 100644
--- a/dist/types-BWScgZQh.mjs
+++ b/dist/types-BWScgZQh.mjs
@@ -1840,7 +1840,7 @@ async function getVersions(spec, options = {}) {
 	return getVersionsForContext(context, options);
 }
 function getVersionsForContext(context, options) {
-	if (isDefaultRegistry(context.registry)) return getVersions$1(context.parsed.raw, toDefaultOptions(options));
+
 	return getVersionsFromRegistry(context, options);
 }
 function toDefaultOptions(options) {
